### PR TITLE
Add Typescript Def File

### PR DESCRIPTION
**Pull request by sjmcdowall**
Add a typescript definition file.  Not sure about the package-lock.json file .. I think it's ok?

---
This change is [Reviewable](https://reviewable.io/reviews/oleg-koval/koa-verto/33)

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Type definitions for koa-verto 0.0
+
+import * as Koa from 'koa';
+
+export = setVersionHeaders;
+
+declare function setVersionHeaders(options?: verto.Options): Koa.Middleware;
```
Comment (oleg-koval) [inline index.d.ts]: I've never worked with TS, so for me, this is bit unclear, but I don't understand why this function is here, as there is no such function in the original codebase (master branch).

Comment (sjmcdowall) [reply index.d.ts]: OK -- commit message changed with feat:  to the keyword..

Cheers

**Comment by oleg-koval** [comment]
@sjmcdowall 
Do we need to update a Readme about TS file?

**Comment by sjmcdowall** [comment]
I don't think so -- a TS file is just a definition file .. not a big deal and nothing to mention .. at least I haven't seen people mention them before in READMEs :)

**Comment by oleg-koval** [comment]
so I've fixed all blockers,
can you please rebase now on fresh master (v.1.1.0) 
and then I'll do the final review :)

Sorry for all confusions and thanks for the contribution! :)

**Comment by sjmcdowall** [comment]
OK -- I did the rebase and pushed .. that should auto-updated the PR right?

**Pull request merged**